src/libs/Network/NetworkStore.ts

```diff
@@ -53,17 +49,17 @@ function resetHasReadRequiredDataFromStorage() {
 Onyx.connect({
     key: ONYXKEYS.SESSION,
     callback: (val) => {
-        authToken = lodashGet(val, 'authToken', null);
-        supportAuthToken = lodashGet(val, 'supportAuthToken', null);
-        currentUserEmail = lodashGet(val, 'email', null);
+        authToken = val?.authToken ?? null;
+        supportAuthToken = val?.supportAuthToken ?? null;
+        currentUserEmail = val?.email ?? null;
         checkRequiredData();
     },
 });
 
 Onyx.connect({
     key: ONYXKEYS.CREDENTIALS,
     callback: (val) => {
-        credentials = val || {};
+        credentials = val;
```

Comment:
Why did you remove the `|| {}` wasn't that in place so that we don't have `null` in `credentials`?

Reply:
Because credentials are of type `Credentials | null`, and `{}` is not allowed

Reply:
Also `val` is of type `Credentials` | `null`